source/language/openpulse.rst

```diff
@@ -0,0 +1,609 @@
+OpenPulse Grammar
+=================
+
+OpenQASM allows users to provide the target system's implementation of quantum operations
+with ``defcal`` blocks . Calibration grammars are open to extension for system implementors. In
+this document, we outline one such grammar, OpenPulse, which maybe selected within a supporting
+compiler through the declaration defcalgrammar "openpulse" <VERSION>.
+
+This grammar is motivated by the original OpenPulse specification, a JSON wire-format for
+pulse-level quantum programs defined in the paper `Qiskit Backend Specifications for OpenQASM
+and OpenPulse Experiment`_:cite:`mckay2018`. The textual format described
+here has several advantages over the original JSON format:
+
+- Improved readability
```

Comment:
Extremely minor but should be consistent on periods vs no periods at the end of the bullets

Reply:
Fixed in upcoming 597c3e5.